sky/provision/kubernetes/utils.py

```diff
@@ -2395,3 +2396,23 @@ def process_skypilot_pods(
         num_pods = len(cluster.pods)
         cluster.resources_str = f'{num_pods}x {cluster.resources}'
     return list(clusters.values()), jobs_controllers, serve_controllers
+
+
+def get_gpu_resource_key():
```

Comment:
Can we also update L194 in sky/provision/kubernetes/instance.py to use get_gpu_resource_key() to make sure errors are handled correctly?
https://github.com/skypilot-org/skypilot/blob/c1726ae31068da1b8faba5c1b605ebc0f9fb2f35/sky/provision/kubernetes/instance.py#L194

Reply:
fixed